Tests/TestUtilities/TestLogger.cs

```diff
@@ -247,15 +249,11 @@ public void WriteUIWarnings(string outputFolder)
         // no-op
     }
 
-    public void AddTelemetryMessage(string key, object value)
-    {
-        // no-op
-    }
+    public void AddTelemetryMessage(string key, object value) =>
+        consoleLogger.AddTelemetryMessage(key, value);
```

Comment:
Just add a `List<KeyValuePair<string, object>> TelemetryMessages` property like for the other Messages.